### PR TITLE
Add scrollTo signatures to support scrolling on custom view matcher.

### PR DESCRIPTION
**Pull request by glureau-betclic**
This is quite convenient when we have UI structure a bit more complex than a unique ID/string.

### CODE REVIEW

File: library/src/main/java/com/schibsted/spain/barista/interaction/BaristaScrollInteractions.kt [hunk 1]
```diff
@@ -16,43 +16,53 @@ import org.hamcrest.Matcher
...
+        scrollWithMultipleAttempts(id.resourceMatcher(), failAtEnd = true)
+    }
+
+    @JvmStatic
+    fun scrollTo(text: String) {
+        scrollWithMultipleAttempts(withText(text), failAtEnd = true)
+    }
+
+    @JvmStatic
+    fun safelyScrollTo(matcher: Matcher<View>) {
```
Comment (rocboronat) [inline library/src/main/java/com/schibsted/spain/barista/interaction/BaristaScrollInteractions.kt]: @AdevintaSpain/barista-maintainers we should explain this method in the README or at least write a Javadoc for it...

**Comment by rocboronat** [comment]
@alorma @Sloy @EsteveAguilera any feedback? IMHO we could merge it now, but I don't want to be the only eyes there. Looking for a naughty guy to dance with! 👯‍♂

**Comment by alorma** [comment]
Go for it @rocboronat!

**Comment by EsteveAguilera** [comment]
LGTM!

**Pull request merged**